config/environments/development.rb

```diff
@@ -79,4 +79,7 @@ def interesting_methods
       self.public_methods.sort - Object.new.public_methods
     end
   end
+  def interesting_methods_filter(where)
```

Comment:
I'm going to need some clarification on why this method exists...as a casual reader, I don't understand this one.

Reply:
Okay. I'll give an example.  Basically, the object.interesting_methods returns a lot of methods.  If I know part of the method name that I'm looking for, this allows me to pass what I know (that gets regex matched *what_i_know* to filter the other object.interesting_methods that I'm not really interested it.